app/controllers/tag_controller.rb

```diff
@@ -114,6 +115,16 @@ def show
                 .where('term_data.name = ?', params[:id])
     @length = Tag.contributor_count(params[:id]) || 0
 
+    # fetching nodes with revisions for contributor tab data
+    @tagnames = [params[:id]]
+    @tag = Tag.find_by(name: params[:id])
+    @notes2 = Node.where(status: 1, type: 'note')
```

Comment:
Here, do we need to name this `@noteCount` and pass on the `.count` of this rather than the whole notes list?

Let's also leave a comment here to `# later, move this into a method in the Tag model like Tag.taggedNodeCount(tagname)` -- what do you think? could be a modification of this maybe?

https://github.com/publiclab/plots2/blob/master/app/models/tag.rb#L85

Reply:
I have made function also as you stated, and other changes are also done.